src/main/java/lib/controllers/GameController.java

```diff
@@ -85,6 +85,10 @@ public Trigger get(Button button) {
     return new Trigger(() -> m_controller.getRawButton(button.id));
   }
 
+  public Trigger get(BooleanSupplier booleanSupplier) {
+    return new Trigger(() -> booleanSupplier.getAsBoolean());
+  }
+
```

Comment:
I don't get why this method you can just do "new Trigger(boleanSupplier)" the need to make method in game controller is riduclus. Similarly you don't need to convert a boolean supllier into a boolean supplier by makeing it a lambda.

Reply:
It's easier to use m_operator.get() than new Trigger(), because that is what the rest of the code uses. The lambda is for getting as a boolean. All of the other methods in the class use it, too. This also used to exist, so I just re-added it.

Reply:
A Boolean supplier supplies a Boolean and therefore you don't need to get as a Boolean because it already does that and therefore a lambda is repetitive.